editor/osm_feature_matcher.cpp

```diff
@@ -128,9 +127,12 @@ double ScoreGeometry(pugi::xml_document const & osmResponse,
     }
   }
 
-  auto const wayScore = static_cast<double>(matched) / theirGeometry.size() - 0.5;
-  auto const geomScore = static_cast<double>(matched) / ourGeometry.size() - 0.5;
-  auto const result = wayScore <= 0 || geomScore <= 0
+  auto const wayScore = static_cast<double>(matched) / theirGeometry.size();
+  auto const geomScore = static_cast<double>(matched) / ourGeometry.size();
```

Comment:
Can we use for geomScore the intersection of expected and actual shapes divided by union of expected and actual shapes?